UnityProject/Assets/Scripts/Electricity/FunctionsAndClasses/PowerSupplyFunction.cs

```diff
@@ -14,6 +14,12 @@ public static class PowerSupplyFunction  {
 	/// <param name="Supply">The supplying device that is turned off</param>
 	public static void TurnOffSupply(ModuleSupplyingDevice Supply)
 	{
+		if (Supply.ControllingNode == null)
+		{
+			Logger.LogError("Supply.ControllingNode == null");
+			return;
+		}
+
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
PowerSupplyFunction.TurnOffSupply (ModuleSupplyingDevice Supply) (at /github/workspace/UnityProject/Assets/Scripts/Electricity/FunctionsAndClasses/PowerSupplyFunction.cs:10)
ModuleSupplyingDevice.TurnOffSupply () (at /github/workspace/UnityProject/Assets/Scripts/Electricity/NodeModules/ModuleSupplyingDevice.cs:118)
ReactorTurbine.OnDisable () (at /github/workspace/UnityProject/Assets/Scripts/Electricity/Engine/ReactorTurbine.cs:33)